### PR TITLE
Feature/dfe 625 update error messages

### PR DESCRIPTION
**Pull request by bennettstuart**
https://alpha-dfe-data.atlassian.net/browse/DFE-625

### CODE REVIEW

File: src/explore-education-statistics-frontend/src/modules/table-tool/components/FiltersForm.tsx [hunk 1]
```diff
@@ -52,6 +52,16 @@ const FiltersForm = (props: Props & InjectedWizardProps) => {
     <WizardStepHeading {...props}>Choose your filters</WizardStepHeading>
   );
 
+  function getSectionError(errorLabel: string, spliceOut: string) {
```
Comment (ntsim) [inline src/explore-education-statistics-frontend/src/modules/table-tool/components/FiltersForm.tsx]: Why are we doing this? GDS error message guidance states that the error messages must be the same between the error summary and the input error.

https://design-system.service.gov.uk/components/error-summary/#how-it-works

Comment (ntsim) [inline src/explore-education-statistics-frontend/src/modules/table-tool/components/FiltersForm.tsx]: Why are we doing this? GDS guidance on the Error summary component states that the error messages should be the same as the field input.

https://design-system.service.gov.uk/components/error-summary/#how-it-works

Comment (bennettstuart) [reply src/explore-education-statistics-frontend/src/modules/table-tool/components/FiltersForm.tsx]: Following Fred's comments on the ticket

> Change error message content under ‘There is a problem’ to match following template (as it provides user with greater context as to where error is occuring on page below):
> 
> [NAME OF SECTION] - select at least one
> 
> For example:
> 
> Indicator - select at least one
> School type - select at least one
> Characteristic - select at least one 
> 
> AND 
> 
> Academy - select at least one option
> All through school flag - select at least one option
> 
> etc etc for each section. 
> 
> Ordering content this way makes it easier to read (as each sentence doesn’t start with the same words) and helps users differentiate where errors are on the table tool below so they can quickly find them.
> 
> Content within each section should then just read as follows as there's no need to add name of section (as that’s already set by the title of the section above) and user just needs to be remind of what they need to do:
> 
> Select at least one option

Comment (ntsim) [reply src/explore-education-statistics-frontend/src/modules/table-tool/components/FiltersForm.tsx]: I would just push back against this one as again, this is breaking the GDS error message pattern.

Comment (bennettstuart) [reply src/explore-education-statistics-frontend/src/modules/table-tool/components/FiltersForm.tsx]: Right-o, so I’ve reverted the error message text changes that were made for the *Choose your filters* section (back to their version defined in dfe-706)
keeping the errors the same in both instances, in the “There is a problem” box and by the input itself as outlined in GDS

https://alpha-dfe-data.atlassian.net/browse/DFE-706

**Pull request merged**